docs/compute/project/createProject.md

```diff
@@ -0,0 +1,69 @@
+# Create a New Project
+You can create a project by clicking on the **Create new project** button in the Projects list page. 
+This will pop-up a dialog, in which you enter the project name and an optional description. You can also select an 
+initial set of members for the project, who will be 
+given the role of Data Scientist in the project. Member roles can later be updated in the Project settings 
+by the project owner or a member with the data owner role. A valid project name can only contain characters 
```

Comment:
Add a link to the multitenant doc like: `for more information about the roles see the documentation here`